samwise.js

```diff
@@ -1,39 +1,37 @@
-const largestNumber = Math.max(...numbers);
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.